api/questions/tests/tests.py

```diff
@@ -0,0 +1,194 @@
+import json
+import responses
+import re
+import os
+import pytz
+
+from rest_framework.test import APITestCase, APIClient
+from rest_framework.reverse import reverse
+from rest_framework import status
+from rest_framework.authtoken.models import Token
+
+from django.contrib.auth import get_user_model
+
+from datetime import datetime
+
+from questions.models import Question
+from meetups.models import Meetup
+from users.models import User
+# Create your tests here.
+
+class BaseTest(APITestCase):
+    """
+    The base where are default test case settings kept
+    """
+
+    def setUp(self):
+        self.client = APIClient()
+        self.user = User.objects._create_user(
+            name="Abraham Kamau",
+            email="admin@questioner.com",
+            password="@Admin123"
+        )
+        self.user.is_active = True
+        self.user.is_admin = True
+        self.user.save()
+
+        self.meetup = Meetup.objects.create(
+            title='Test Driven Development',
+            body='Developers need to discusss the correct approach of doing test driven development',
+            location='Andela Campus',
+            creator=self.user,
+            scheduled_date=pytz.utc.localize(datetime.strptime(
```

Comment:
Hello @a-braham ,
Instead of using pytz and a fixed date you can django's timezone. It will give you a localized time.
```
from django.utils import timezone
scheduled_date=timezone.now()+timezone.timedelta(days=3)
```
will make the meetup to be scheduled 3 days from now and the time is localized.
Having a fixed time would mean that the test will start to fail after the specified date since you can't create a meetup on a time that has passed.

Reply:
I have implemented it. Looks good